images/installer-openstack/Dockerfile

```diff
@@ -1,29 +1,29 @@
-FROM centos:7
+FROM openshift/origin-ansible
 
 MAINTAINER Andrew Block “andrew.block@redhat.com”
 
+USER root
+
 # Update System and install clients
 RUN yum install -y epel-release; \
   yum update -y; \
-  yum install -y python-passlib ansible; \
-  yum install -y --setopt=tsflags=nodocs https://repos.fedorapeople.org/repos/openstack/openstack-ocata/rdo-release-ocata-3.noarch.rpm centos-release-openshift-origin; \
-  yum install -y python-devel git tar gcc libffi-devel openssl-devel bind-utils httpd-tools java-1.8.0-openjdk-headless\
-                 python-pip python-ceilometerclient \
+  yum install -y --setopt=tsflags=nodocs https://repos.fedorapeople.org/repos/openstack/openstack-ocata/rdo-release-ocata-3.noarch.rpm; \
+  yum install -y python-pip python-ceilometerclient \
                  python-cinderclient python-glanceclient \
                  python-heatclient python-neutronclient \
                  python-novaclient python-saharaclient \
                  python-swiftclient python-troveclient \
                  python-openstackclient python-dns \
-                 pyOpenSSL \
-                 origin-clients; \
+                 pyOpenSSL;\
   yum clean all; \
   pip install --upgrade pip jinja2; \
   pip install shade six
 
-ADD bin/start.sh /root/
+COPY images/installer-openstack/root /
```

Comment:
This will call for a very specific build command - let's ensure it's documented in the README.